mlrun/model_monitoring/helpers.py

```diff
@@ -20,7 +19,7 @@
 )
 
 
-def get_model_monitoring_stream_processing_function(
+def init_model_monitoring_stream_processing_function(
```

Comment:
if you already used the full name for all other words in the function I don't see reason not to apply it to all words
```suggestion
def initial_model_monitoring_stream_processing_function(
```